.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ defaults:
 jobs:
   dependents:
     runs-on: ubuntu-latest
-    container: swift:5.2-bionic
```

Comment:
This needs to stay as 5.2 to ensure we don't break 5.2 compatibility

Reply:
Fixed! 

I updated the `maxim-lobanov/setup-xcode` to `v1.2.1` since macOS build won't start.

Reply:
@0xTim 5.2 compatibility is checked by the main Linux workflows in each repo (at least, that's how it's supposed to work) - I prefer to run the dependent checks in a "current"-version container to reflect the most common usage, since those checks don't have a version matrix.